tools/importer-rest-api-specs/components/schema/builder.go

```diff
@@ -401,11 +400,10 @@ func (b Builder) findCreateUpdateReadPayloads(input resourcemanager.TerraformRes
 	out.readPayload = readModel
 	// then find the `Properties` model within this
 	readPropsModelName, readPropsModel := out.getPropertiesModelWithinModel(out.readPayload, b.models)
-	if readPropsModelName == nil || readPropsModel == nil {
-		return nil, fmt.Errorf("couldn't find `Properties` model for Read Payload")
+	if readPropsModelName != nil || readPropsModel != nil {
+		out.readPropertiesModelName = *readPropsModelName
+		out.readPropertiesPayload = *readPropsModel
 	}
```

Comment:
(as above)

Reply:
As discussed this change causes unwanted effects in the existing generated resources, details in #3588